src/unfold/settings.py

```diff
@@ -39,7 +39,7 @@
 }
 
 
-@lru_cache
+@lru_cache(maxsize=None)
```

Comment:
We need to declare `maxsize=None` for Python3.7 to avoid this error :
```
src/unfold/settings.py:43: in <module>
    def get_config(settings_name=None):
/usr/lib/python3.7/functools.py:490: in lru_cache
    raise TypeError('Expected maxsize to be an integer or None')
TypeError: Expected maxsize to be an integer or None
```